composer.json

```diff
@@ -1,6 +1,5 @@
 {
     "name": "phile-cms/phile",
-    "version": "1.6.0",
```

Comment:
In the past we run into composer problems without having a hardcoded version number. I tried to document it on the [release page](https://github.com/PhileCMS/Phile/wiki/%5BDEVELOPER%5D-Release-Guide#update-composer):

>  Without the version string composer will not pickup Phile's version number. It assumes "1.0.0" and will throw an error if you want to install Phile-plugins requiring "Phile >1.0.0".

So this may rise issues.